### DISCUSSION TITLE
2023/12/23/Rust%E7%BC%96%E5%86%99QQ%E6%9C%BA%E5%99%A8%E4%BA%BA/

### DISCUSSION THREAD

**Discussion by giscus** [ExquisiteCore/ExquisiteCore.github.io#10] [General]
# 2023/12/23/Rust%E7%BC%96%E5%86%99QQ%E6%9C%BA%E5%99%A8%E4%BA%BA/

Rust编写QQ机器人  最近用openShamrock搭建了一个QQ机器人就打算用新学的Rust写一个客户端程序  项目地址 GitHub 配置 12345#先创建一个Rust项目cargo new rust_q_botcd rust_q_bot#打开项目code .

https://blog.exquisitecore.xyz/2023/12/23/Rust%E7%BC%96%E5%86%99QQ%E6%9C%BA%E5%99%A8%E4%BA%BA/

**Comment by ExquisiteCore** [comment]
测试
